R/raterAgreement.R

```diff
@@ -48,12 +48,10 @@ raterAgreement <- function(jaspResults, dataset, options) {
 
   weighted <- options[["cohensKappaType"]] == "weighted"
 
-  weightedString <- ifelse(weighted, "Weighted", "Unweighted")
-
   # Create the JASP Table
-  jaspTable <- createJaspTable(title = gettextf("Cohen's %s kappa", weightedString))
+  jaspTable <- createJaspTable(title = gettext("Cohen's kappa"))
   jaspTable$addColumnInfo(name = "ratings", title = gettext("Ratings"), type = "string")
-  jaspTable$addColumnInfo(name = "cKappa", title = gettextf("%s kappa", weightedString), type = "number")
+  jaspTable$addColumnInfo(name = "cKappa", title = gettextf("kappa"), type = "number")
```

Comment:
You removed the dynamic naming, isn't it clearer when the output explicitly states if it is a weighted Kappa or not? Or maybe a footnote instead?

Reply:
Hmm, I thought about this, and it comes down to a design choice. I think that if a user clicks on weighted kappa they already expect to see the weighted kappa in the output, why add extra information that is already implied... But I suppose someone else might see this differently.